cypress/support/pages/product.pageObject.js

```diff
@@ -0,0 +1,17 @@
+/// <reference types="Cypress" />
+
+import PageObject from '../PageObject';
+
+class ProductPageObject extends PageObject {
+  selectProduct(productName) {
+    cy.contains('.hrefch', productName)
+      .click();
+  }
+
+  clickAddToCart() {
+    cy.contains('a', 'Add to cart')
```

Comment:
The use of 'a' tag to select the 'Add to cart' button might not select the desired element if there are more than one 'a' tags on the page. Consider using a more specific selector to select the 'Add to cart' button.

Reply:
done